HHS/Head-Start-TTADP
# [PROD] Remove feature flag from recipient record page

## Description of change

Remove feature flag from the recipient record page

**note** - The flag will still appear in the admin, there is a separate task logged to remove the unused flag. It won't have any effect on the app though.

## How to test

View the app as a non-admin, non-feature flag user. You will be able to see the recipient record search and record pages.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-552


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
